src/gui/models.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) by Hannah von Reth <hannah.vonreth@owncloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+#include "models.h"
+
+#include <QApplication>
+#include <QItemSelectionRange>
+#include <QTextStream>
+
+QString OCC::Models::formatSelection(const QModelIndexList &items)
+{
+    // TODO: fine tune
```

Comment:
I actually meant the column width, which isn't that easy.
```
         Action,                                                            File,                        Folder,  Size,             Account,                Time,
The downloaded file does not match the checksum, it will be resumed. 'd4c9b536a6d082dcef9b96a9956986bd3472196f' != 'adc83b19e793491b1c6ea0fd8b46cd9f32e592fc',                                             ownCloud Manual.pdf,                    ownCloud17,6,4 MB,     admin@localhost,2021-04-01T11:08:17.000Z,